services/121-service/src/registration/enum/custom-data-attributes.ts

```diff
@@ -48,6 +48,8 @@ export class Attribute {
   public id?: number;
   public name: string;
   public type: string;
+  // XXX: can we force this to AnswerTypes? Relevant for Ruben's refactoring?
```

Comment:
It will be refactored to something new in Ruben's refactor. But essentially it will be something like `AnswerTypes | CustomAttributeType`